raiden/tests/unit/api/test_python_api.py

```diff
@@ -156,3 +160,55 @@ def test_api_channel_events(raiden_chain):
         channel_0_1.channel_address, max_block + 1, max_block + 100
     )
     assert not results
+
+
+@pytest.mark.parametrize('blockchain_type', ['geth'])
+@pytest.mark.parametrize('number_of_nodes', [2])
+@pytest.mark.parametrize('number_of_tokens', [1])
+@pytest.mark.parametrize('register_tokens', [True, False])
+@pytest.mark.parametrize('settle_timeout', [600])
+def test_channel_open_close(raiden_network, token_addresses, settle_timeout):
+    app0, app1 = raiden_network
+
+    maker_address = app0.raiden.address
+    taker_address = app1.raiden.address
+
+    token_address = token_addresses[0]
+
+    api0 = RaidenAPI(app0.raiden)
+    api1 = RaidenAPI(app1.raiden)
+    api0.open(
```

Comment:
I thought about that, but I'm using this test also for the case where the tokens are previously registered (`@pytest.mark.parametrize('register_tokens', [True, False])`), so I prefered to not assert it, but could add a double check (assert only if `register_tokens` is `False`)